NameGame/NameGame/UserRecord.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  UserRecord.swift
+//  NameGame
+//
+//  Created by Tammy Le on 6/12/17.
+//  Copyright © 2017 WillowTree Apps. All rights reserved.
+//
+
+import Foundation
+
+/** 
+    Models an error that's thrown if key can't be parsed from JSON dictionary.
+ */
+enum ParsingError: Error {
+    case missingKey(String)
+}
+
+/** 
+    Models a user record.
+ */
+
+struct UserRecord {
+    
+    let id: String
+    let firstName: String
+    let lastName: String
+    let imageUrl: String
+    
+    // Initializer given dictionary.
+    init(dict: [String: Any]) throws {
+        guard
```

Comment:
Style change:

```
guard let id = dict[JSONKey.id.rawValue] as? String else {
    throw ParsingError.missingKey(JSONKey.id.rawValue)
}
```

In most cases you want the curly brace trailing the guard statement. Also a white space between each of your guard statements would be nice.